File: src/algorithms/percentage-letter.js
/**
 * Given a string s and a character letter, returns the percentage of characters
 * in s that equal letter rounded down to the nearest whole percent.
 * 
 * @param {string} s - A string of characters
 * @param {string} letter - A string character
 * @returns {number} - Returns the percentage of chars in s that equal letter
 */
function percentageLetter (s, letter) {
  let count = 0;
  for (let i = 0; i < s.length; i++) {
    if (s[i] === letter) {
      count ++;
    }
  }
  return Math.floor((count / s.length) * 100);
}

File: src/algorithms/percentage-letter.py
def percentage_letter(s, letter):
    """Given a string s and a character letter, return the percentage of
    characters in s that equal letter rounded down to the nearest whole percent.

    Args:
        s (String): A string of lowercase English letters.
        letter (String): A lowercase English letter.

    Returns:
        Integer: Returns the percentage that letter appears in s.
    """
    count = 0
    for char in s:
        if char == letter:
            count += 1
    if count == 0:
        return 0
    return count * 100 // len(s)
